cli/tests/integration/check_tests.rs

```diff
@@ -43,3 +43,9 @@ itest!(declaration_header_file_with_no_exports {
   args: "check --quiet declaration_header_file_with_no_exports.ts",
   output_str: Some(""),
 });
+
+itest!(error_mutually_exclusive_remote_args {
+  args: "check --remote --no-remote check_all.ts",
+  output: "check_exclusive_args.out",
+  exit_code: 1,
+});
```

Comment:
It would be preferable to create a unity test in `cli/flags.rs` that asserts error is returned when parsing such flag combination (because unit tests are much faster than integration tests that spawn a subprocess).

Reply:
Changed it to a unit test.